CONTRIBUTING.md

```diff
@@ -206,15 +206,15 @@ You must have the following installed on your system to contribute locally:
 
 #### Debian/Ubuntu
 
-`sudo apt install g++ make` meets the additional requirements to run `node-gyp` in the context of building Cypress from source.
+`sudo apt install build-essential` meets the additional requirements to run `node-gyp` in the context of building Cypress from source.
 `python` is pre-installed on Debian-based systems including Ubuntu.
 The Python versions shipped with Ubuntu versions `20.04`, `22.04` and `24.*` are compatible with Cypress requirements.
 
 For Ubuntu `24.04` and above, refer also to the [Ubuntu 24.04 Release notes](https://discourse.ubuntu.com/t/noble-numbat-release-notes/39890) in the section "Unprivileged user namespace restrictions" and apply one of the workarounds to disable unprivileged user namespace restrictions for the entire system, either for one boot or persistently, as described. If you do not do this you may receive an error which includes the text `FATAL:setuid_sandbox_host.cc` when you try to run Cypress on these versions of Ubuntu after building Cypress from source.
 
 #### Windows
 
-Currently no additional instructions for installation requirements.
+Sometimes, when using [`nvm`](https://github.com/coreybutler/nvm-windows) and changing node versions, node install tools are not installed automatically. This is needed for `node-gyp` to rebuild `better-sqlite3`. If you are failing to build Cypress because of this, either with a Python install missing or a VSCode version not being detected by `node-gyp`, you likely need to run the `install_tools.bat` outlined in the [better-sqlite3 troubleshooting guide](https://github.com/WiseLibs/better-sqlite3/blob/master/docs/troubleshooting.md).
```

Comment:
I don't think this can be the final suggestion for Windows, as `install_tools.bat` installs the old Visual Studio 2019 and when I have only this version installed, then the rebuild fails.

The old text isn't right either though, so I would consider this just a WIP at this time. There are multiple related compatibility issues concerning Node.js, node-gyp, python3, Visual Studio 2019 / 2022, registry-js and @cypress/get-windows-proxy.

I suggest to revisit after the release of Cypress 15. I will try to identify the individual issues and write them up in due course.

Reply:
@AtofStryker 

- After the merge of https://github.com/cypress-io/cypress/pull/32019 it is possible to rebuild Cypress 15 with either Visual Studio 2019 or 2022, so it's OK to leave the text as it is. Except the introductory "Sometimes, ..." is misplaced because nvm-windows **never** installs Python + Visual Studio. 

https://github.com/nodejs/node-gyp/blob/main/README.md#on-windows is the reference documentation to be using